src/lib/create-utils.js

```diff
@@ -0,0 +1,129 @@
+const inquirer = require('inquirer');
+const actorTemplates = require('@apify/actor-templates');
+const { validateActorName } = require('./utils');
+
+const PROGRAMMING_LANGUAGES = ['JavaScript', 'TypeScript', 'Python'];
+
+/**
+ * @param {string} maybeActorName
+ * @returns {Promise<string>}
+ */
+exports.ensureValidActorName = async (maybeActorName) => {
+    if (maybeActorName) {
+        validateActorName(maybeActorName);
+        return maybeActorName;
+    }
+    return promptActorName();
+};
+
+/**
+ * @param {string} maybeTemplateName
+ * @returns {Promise<object>}
+ */
+exports.getTemplateDefinition = async (maybeTemplateName) => {
+    const manifest = await actorTemplates.fetchManifest();
+    if (maybeTemplateName) {
+        return manifest.templates.find((t) => t.name === maybeTemplateName);
+    }
+
+    const programmingLanguage = await promptProgrammingLanguage();
+    while (true) {
+        const templateDefinition = await promptTemplateDefinition(manifest, programmingLanguage);
+        const shouldInstall = await promptTemplateInstallation(templateDefinition);
+        if (shouldInstall) return templateDefinition;
+    }
+};
+
+/**
+ * @returns {Promise<string>}
+ */
+async function promptActorName() {
+    const answer = await inquirer.prompt([{
+        name: 'actorName',
+        message: 'Name of your new actor:',
+        type: 'input',
+        validate: (promptText) => {
+            try {
+                validateActorName(promptText);
+            } catch (err) {
+                return err.message;
+            }
+            return true;
+        },
+    }]);
+    return answer.actorName;
+}
+
+/**
+ * @returns {Promise<string>}
+ */
+async function promptProgrammingLanguage() {
+    const choices = PROGRAMMING_LANGUAGES.map((lang) => ({
+        name: `👉 ${lang}`,
```

Comment:
I'm not a fan of these 👉 emojis, you already have the `>` indicator there from inquirer, so it's kinda duplicate.

Reply:
Yeah, I also felt those are a little bit too much. Will remove them.